examples/load_balancer/load_balancer.c

```diff
@@ -555,6 +628,7 @@ packet_handler(struct rte_mbuf *pkt, struct onvm_pkt_meta *meta,
                 for (i = 0; i < RTE_ETHER_ADDR_LEN; i++) {
                         flow_info->s_addr_bytes[i] = ehdr->s_addr.addr_bytes[i];
                 }
+                if(debug_mode) printf("New connection made with server %d.\n",flow_info->dest);
```

Comment:
Can we have this display the src IP and port? Something like:
`192.168.1.100:4256 assigned to server 2`